onnx_caffe2/helper.py

```diff
@@ -3,9 +3,62 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import contextlib
+
+from caffe2.python import workspace
 from onnx import helper
+from onnx.backend.base import namedtupledict
 
 
 def make_model(graph, **kwargs):
     kwargs.setdefault('producer_name', 'onnx-caffe2')
     return helper.make_model(graph=graph, **kwargs)
+
+
+@contextlib.contextmanager
+def _caffe2_workspace(name=b"caffe2_ws"):
```

Comment:
This is old code, but I noticed it when reading the diff. We should consider using the `Workspace` class from backend: move it to a separate module.

Reply:
Good catch. Will update.